recipes/ios-cmake/all/conanfile.py

```diff
@@ -0,0 +1,142 @@
+    exports_sources =  "cmake-wrapper"
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    @staticmethod
+    def _chmod_plus_x(filename):
+        if os.name == 'posix':
+            os.chmod(filename, os.stat(filename).st_mode | 0o111)
+
+    def configure(self):
+        if not tools.is_apple_os(self.settings.os):
+            raise ConanInvalidConfiguration("This package only supports Apple operating systems")
+
+    def _guess_toolchain_target(self, os, arch):
+        if os == "iOS":
+            if arch in ["armv8", "armv8.3"]:
+                return "OS64"
+            if arch == "x86_64":
+                return "SIMULATOR64"
+            # 32bit is dead, don't care
+        elif os == "watchOS":
+            if arch == "x86_64":
+                return "SIMULATOR_WATCHOS"
+            else:
+                return "WATCHOS"
+        elif os == "tvOS":
+            if arch == "x86_64":
+                return "TVOS"
+            else:
+                return "SIMULATOR_TVOS"
+        raise ConanInvalidConfiguration("Can not guess toolchain_target. Please set the option explicit (or check our os settings)")
+
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ios-cmake-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        pass # there is nothing to build
+
+    def package(self):
+        self.copy("cmake-wrapper", dst="bin")
+        self.copy("ios.toolchain.cmake",
+                    src=self._source_subfolder,
+                    dst=os.path.join("lib", "cmake", "ios-cmake"),
+                    keep_path=False)
+        self._chmod_plus_x(os.path.join(self.package_folder, "bin", "cmake-wrapper"))
+
+        self.copy("LICENSE.md", dst="licenses", src=self._source_subfolder, keep_path=False)
+        # satisfy KB-H014 (header_only recipes require headers)
+        tools.save(os.path.join(self.package_folder, "include", "dummy_header.h"), "\n")
+
+    def package_info(self):
+        if self.settings.os == "Macos":
+            if not getattr(self, "settings_target", None):
+                #  not a build_require , but can be fine since its build as a ppr:b, but nothing to do
+                return
+            # this is where I want to be, expecting this as a build_require for a host
+            target_os = str(self.settings_target.os)
+            arch_flag = self.settings_target.arch
+            target_version= self.settings_target.os.version
+        elif self.settings.os == "iOS": # old style 1 profile, don't use
+            target_os = str(self.settings.os)
+            arch_flag = self.settings.arch
+            target_version = self.settings.os.version
+        else:
+            #hackingtosh ? hu
+            raise ConanInvalidConfiguration("Building for iOS on a non Mac platform? Please tell me how!")
```

Comment:
```suggestion
            raise ConanInvalidConfiguration("Building for iOS on a non Mac platform? Open an issue.")
```

Nice comment, just try to point to the correct place.